### PR TITLE
Task: How to crawl a web page using python

### LINKED ISSUE #6: Understand how to crawl a web page using Python
the purpose of this task or issue is to understand how you can access a website in python script.

following points need to be taken care in this task:

- how to send url request via python script
- how to grab data recieved from url request
- how to conveniently parse the data and extract relevant information

this task does not expect any scripts to be added in the repository but the expectation will be that you understand how you can retrieve and access data from www.

### PR DESCRIPTION
**Pull request by Raspberry-beans**
In this task we understand that how we can crawl a web page using python urllib function. For further extracting relevant 
        information, we parse variables to grab that information.

Relates : 6

### CODE REVIEW

**Comment by Raspberry-beans** [comment]
Hi @M-Shan 
Task 6 has been completed. Do let me know if further changes are required!

**Comment by Raspberry-beans** [comment]
@M-Shan  Please Update !

File: projects/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Projects
+## This file contains useful information regarding current projects
```
Comment (M-Shan) [inline projects/README.md]: `This file` --> `This folder`

File: projects/news_feed/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+You can check the status of project by clicking on [M-Shan TechCrunch projects](https://github.com/M-Shan/TechCrunch/projects)
+
+### How to access a website using python
+
+> The first step is to understand that how we can send URL requests using python. Luckily, Python comes with the package called **urllib** that simplifies this task. This package further contains five modules.
+1. Request
+1. Response
+1. Error
+1. Parse
+1. Robotparser
```
Comment (M-Shan) [inline projects/news_feed/README.md]: good info but you could also check some high level python modules (pandas, beautifulsoup) for web page crawling.
check following links:

- https://www.edureka.co/blog/web-scraping-with-python/
- https://realpython.com/beautiful-soup-web-scraper-python/

these are just suggestions from my side you can decide what to use in future :)

**Review by M-Shan — CHANGES_REQUESTED** [review]

File: projects/news_feed/README.md (hunk 2 above)
Comment (Raspberry-beans) [reply projects/news_feed/README.md]: @M-Shan README.md files have been updated. I have read one of the links that you shared. They are quite interesting and helpful!

**Review by M-Shan — APPROVED** [review]
Good work.

Check next task.

**Pull request merged**